Add Containers Page (#305)
rancher-sandbox/docs.rancherdesktop.io: docs/ui/containers.md

<!--- Insert S3 Image Here Once Uploaded -->
<Tabs groupId="os" defaultValue={TabsConstants.defaultOs}>
<TabItem value="Windows">

Windows_Image_TBD

</TabItem>
<TabItem value="macOS">

![Containers_Example](../img/Containers_Example.png)

</TabItem>
<TabItem value="Linux">

Linux_Image_TBD

</TabItem>
</Tabs>

The **Containers** tab offers quick access to manage containers and view key information such as:

 - `State`:
  The container state(s) will be listed in this field, and by default running containers are presented first.
- `Name`:
  The container name(s) will be listed in this field and can be sorted.
- `Image`:
  All image names will be listed in this field and can be sorted.
- `Port(s)`:
  Ports are listed in this field and can be clicked for quick access to a localhost port.
- `Started`:
  Container start-up times will be listed in this field and can be sorted.

The listed information can be sorted in ascending or descending order. Containers can also be filtered with input text in the `Filter` field located at the top right. Bulk selection is available for managing multiple instances at once.

### Container Management

> These buttons are located at the top of the page and allow you to perform the following actions below only after selecting a container. Upon selection, the button(s) will be highlighted if the action is applicable to your container state. Bulk selection and actions can also be performed on instances with the same state.

Reviewer: ```suggestion
The buttons located at the top of the page will be highlighted when a container is selected and depend on the container's state.

Bulk selection and actions can also be performed on instances with the same state.

The following actions are available:
```
